src/init/ossec-server.sh

```diff
@@ -345,6 +374,25 @@ start()
                 ${DIR}/bin/${i} ${DEBUG_CLI};
             fi
             if [ $? != 0 ]; then
+                failed=true
+            else
+                is_optional ${i};
+                if [ $? = 0 ]; then
+                    j=0;
+                    while [ $failed = false ]; do
```

Comment:
The while iterates 10 times until a valid attempt is found. I think it would be for the best not to use the loop here. Even if the process dies just once, that is enough to mark it as a fail.  Perhaps the sleep should be increased to 1 sec. 
Lastly, according to line **428**, sleep doesn't work in AIX.